pyproject.toml

```diff
@@ -1,23 +1,24 @@
 [tool.poetry]
-name = "data-product-definition-tooling"
-version = "0.0.13"
-description = "Data Product Definition Tooling"
+name = "ioxio-data-product-definition-tooling"
+version = "0.0.1"
```

Comment:
I'm not sure I like the version number moving backwards despite a rename of the project. I'd actually suggest calling this `0.1.0` instead.